ansible_events/websocket.py

```diff
@@ -1,22 +1,28 @@
 import json
 import logging
+from asyncio.exceptions import CancelledError
 
 import websockets
 
 
 async def send_event_log_to_websocket(event_log, websocket_address):
-    logger = logging.getLogger()
+    logger = logging.getLogger("ansible_events.websocket")
```

Comment:
Logger can be initialized globally. No need to perform a lookup at every function call.